### ISSUE TITLE
Access the softmax value of policy network

### ISSUE THREAD

**Issue by rsanchezgarc** [tensorforce/tensorforce#780]
Hi,

I was wondering if it is possible to obtain as continuous actions the last layer of the network (logits) instead of the argmax over the logits. 

Thank you very much.

Ruben

**Comment by AlexKuhnle** [comment]
Hi, setting the `tracking` argument to `'all'` ([docs](https://tensorforce.readthedocs.io/en/latest/agents/tensorforce.html)) and then accessing the tracked values via `agent.tracked_tensors()` ([docs](https://tensorforce.readthedocs.io/en/latest/agents/tensorforce.html)) should hopefully work for this.

**Comment by rsanchezgarc** [comment]
HI,

This was what I needed.

Thanks a lot!

Ruben

**Comment by AlexKuhnle** [comment]
Great, closing this then.
